### DISCUSSION TITLE
eMylo Matter Thread ZigBee 3.0 Gateway

### DISCUSSION THREAD

**Discussion by gianlucaf81** [Koenkk/zigbee2mqtt#23019] [Devices]
I have this device, it works with tuya, but I couldn't get it to work with z2m, do you have any suggestions?
Model: THP10-Z-X
![448175818_25635895866058239_5151548051159817014_n](https://github.com/Koenkk/zigbee2mqtt/assets/39622729/6b6c2ed9-2966-4116-babc-e430d83594dd)

**Comment by francisp2** [comment]
You can hope it is the same internal as the lidl/silvercrest gateway, and take this approach :  

https://paulbanks.org/projects/lidl-zigbee/

**Reply by raenye** [reply]
There's a Sigmastar SSD202D* dual 1.2GHz ARM SoC inside with a 128MiB NAND flash. Powerful enough for OpenWrt if you ask me :)

See some discussion here: https://github.com/zigpy/zigpy/discussions/650#discussioncomment-7793958

* Somewhat weird choice, given the fact there's no use to the video decoding abilities.
